src/conversion_helper.js

```diff
@@ -758,34 +780,66 @@ function filterConversions(potentialConversions) {
 /*
   Input: imperial conversions
     [
-      { "imperial" : { "number" : 10000, "unit" : " miles" } },
-      { "imperial" : { "number" : 30, "unit" : " mpg" } }
+      { "imperial" : { "numbers" : 10000, "unit" : " miles" } },
+      { "imperial" : { "numbers" : 30, "unit" : " mpg" } }
     ]
   Output: metric and imperial conversions
     [
       { 
         "imperial" : 
-          { "number" : 10000, "unit" : " miles" }, 
+          { "numbers" : 10000, "unit" : " miles" }, 
         "metric": 
-          { "number" : 16093.44, "unit" : " km" } 
+          { "numbers" : 16093.44, "unit" : " km" } 
       },
       { 
         "imperial" : 
-          { "number" : 30, "unit" : " mpg" }, 
+          { "numbers" : 30, "unit" : " mpg" }, 
         "metric" : [
-          { "number" : 12.7543, "unit" : " km/L" },
-          { "number" : 7.84049, "unit" : " L/100km" } 
+          { "numbers" : 12.7543, "unit" : " km/L" },
+          { "numbers" : 7.84049, "unit" : " L/100km" } 
         ]
       }
     ]
 */
 function calculateMetric(imperialInputs) {
   return imperialInputs.map(input => {
     const imperialUnit = input['imperial']['unit'];
-    const imperialNumber = Number(input['imperial']['number']);
+    const imperialNumbers = input['imperial']['numbers'];
 
     const map = unitLookupMap[imperialUnit];
-    input['metric'] =  map['conversionFunction'](imperialNumber)
+    inRangeVal = -Infinity;
+    let multipleResults = false;
+    imperialNumbers.forEach(function(item) {
+        var tempMap = map['conversionFunction'](Number(item));
```

Comment:
This can be declared as `const` because it is not changing values (same comment for the `tempMap`s below)